src/metabase/public_settings.clj

```diff
@@ -82,6 +82,26 @@
   :setter     :none
   :getter     #(uuid-nonce :analytics-uuid))
 
+(defn- first-user-creation
+  "Returns the timestamp at which the first user was created."
+  []
+  (:min (db/select-one ['User [:%min.date_joined :min]])))
+
+(defsetting instance-creation
+  (deferred-tru "The approximate timestamp at which this instance of Metabase was created, for inclusion in analytics.")
+  :visibility :public
```

Comment:
Should this be publicly visible? `:public` means anybody, even people who aren't authenticated, can see this value

Reply:
Yeah, since some analytics events are sent during the initial setup flow, before any user accounts have been created.

Do you see any risk for this value being public? If so maybe we can find a workaround.

Reply:
I don't think it's _particularly_ sensitive information. The only thing I can think of is that exposing instance age might make it easier for bad actors to identify potentially higher-value long-running instances in the wild. Suppose I wanted to set up some honeypot MB instances as a security measure... maybe the bad actors could tell which ones are the honeypots because they are newer then the one where I keep all the big secrets. 

But in terms of weighing the 0.001% security decrease vs. enabling the new analytics I think it's totally fine in this case. Just wanted to make sure we were doing it for a good reason

Reply:
Cool, will keep it public then.